components/Checkbox/Checkbox.jsx

```diff
@@ -18,9 +18,17 @@ const CheckboxWrapper = styled.div`
 `;
 
 const CheckboxLabel = styled(Label)`
-  color: ${Colors.BLACK['700']};
+  color: ${({
+    theme: {
+      colors: { neutral },
+    },
+  }) => neutral[700]};
   font-size: 16px;
-  margin: 0 0 0 10px;
+  margin: ${({
+    theme: {
+      spacing: { xsmall },
+    },
+  }) => `0 0 0 ${xsmall}px`};
 `;
```

Comment:
What do you think to put the color and margin callbacks together?

```suggestion
const CheckboxLabel = styled(Label)`
  ${({ theme: { colors: { neutral }, spacing: { xsmall } } }) => `
    color: ${neutral[700]};
    margin: 0 0 0 ${xsmall}px;
  `}
  font-size: 16px;
`;
```

Reply:
And baseFontSize too (if there's not a reason)